examples/sparse/gcnii.py

```diff
@@ -0,0 +1,172 @@
+"""
+[Simple and Deep Graph Convolutional Networks]
+(https://arxiv.org/abs/2007.02133)
+"""
+
+import math
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from dgl.data import CoraGraphDataset
+from dgl.mock_sparse import create_from_coo, diag, identity
+from torch.optim import Adam
+
+
+class GCNIIConvolution(nn.Module):
+    def __init__(self, in_size, out_size):
+        super().__init__()
+        self.out_size = out_size
+        self.weight = nn.Linear(in_size, out_size, bias=False)
+
+    ############################################################################
+    # (HIGHLIGHT) Take the advantage of DGL sparse APIs to implement the GCNII
+    # forward process.
+    ############################################################################
+    def forward(self, A_norm, H, H0, lamda, alpha, l):
+        beta = math.log(lamda / l + 1)
+
+        # Multiply a sparse matrix by a dense matrix.
+        H = A_norm @ H
+        support = (1 - alpha) * H + alpha * H0
+        H = (1 - beta) * support + beta * self.weight(support)
+        return H
+
+
+class GCNII(nn.Module):
+    def __init__(
+        self,
+        in_size,
+        out_size,
+        hidden_size,
+        n_layers,
+        lamda,
+        alpha,
+        dropout=0.5,
+    ):
+        super().__init__()
+        self.hidden_size = hidden_size
+        self.n_layers = n_layers
+        self.lamda = lamda
+        self.alpha = alpha
+
+        self.FC_layers = nn.ModuleList()
```

Comment:
what does FC stand for?

Reply:
in the paper, it first uses a FC layer to convert the init features into fixed size, then use GCNIIConv